doc/conf/Doxyfile.in

```diff
@@ -79,29 +85,33 @@ GRAPHICAL_HIERARCHY    = YES
 DIRECTORY_GRAPH        = YES
 DOT_IMAGE_FORMAT       = svg
 INTERACTIVE_SVG        = YES
+GENERATE_LEGEND        = YES
+HAVE_DOT               = YES
+TEMPLATE_RELATIONS     = YES
+INCLUDE_GRAPH          = YES
+INCLUDED_BY_GRAPH      = YES
+MAX_DOT_GRAPH_DEPTH    = 5
+
+DOT_CLEANUP            = YES
 
 # HTML-specific configuration
 HTML_OUTPUT            = .
 HTML_FILE_EXTENSION    = .html
 HTML_HEADER            =
-HTML_FOOTER            =
-HTML_STYLESHEET        =
-HTML_EXTRA_STYLESHEET  =
-HTML_EXTRA_FILES       =
+HTML_FOOTER            = 
 HTML_COLORSTYLE_HUE    = 220
-HTML_COLORSTYLE_SAT    = 100
-HTML_COLORSTYLE_GAMMA  = 80
-HTML_TIMESTAMP         = NO
-HTML_DYNAMIC_SECTIONS  = NO
-HTML_INDEX_NUM_ENTRIES = 100
+HTML_COLORSTYLE_SAT    = 200
+LAYOUT_FILE            = @CMAKE_CURRENT_SOURCE_DIR@/DoxygenLayout.xml
+HTML_EXTRA_STYLESHEET  = @CMAKE_CURRENT_SOURCE_DIR@/stylesheet.css
 
 # LaTeX-specific configuration
+GENERATE_LATEX         = NO
```

Comment:
Is there a reason for not generating Latex? This could be a useful documentation format for some people (you can do control-f for example).

Reply:
This is to generate a pdf, I think. It has been enabled in the pdf config file.

Reply:
Ah right, there is a specific PDF doxyfile. Then, my question would be, we probably should also generate the pdf and publish it onto tho gh-pages branch so that the documentation is accessible both as html and pdf by default.

I'm unsure of the state of the PDF actually generated with your current changes though. If it looks terrible and requires several changes, forget it for now, we could take care of this in another PR.

Reply:
Yes, unfortunately, the pdf is not in a good state.

Reply:
I just tried it out and it's actually quite decent. Here is an example pdf file: https://ufile.io/cccax

The changes I had to do for that are the following. I actually generated that without even using the `Doxyfile-pdf.in`, but if that uses a similar configuration it should come out similarly. The main change for the documentation to work is to strip all `\f` in Linop documentation. Then tweaking a little the Doxyfile.
https://pastebin.com/AsF1BFdx

To generate the pdf, the following process has to be done manually:
```bash
cd doc/usr
make
# Afterwards, everything is available in refman.pdf
```

Reply:
I like it!

Reply:
Yes, thanks. I had a problem with hyperlinks. But I think `COMPACT_LATEX=NO` fixes that. Also, when you set the `GINKGO_DOC_GENERATE_PDF=on`, it generates the pdf in the doc folder automatically.

Reply:
If you use the `Doxyfile-pdf.in`, no graphs are included (and in fact you would regenerate the graphs twice in addition). So this would at least require some configuration tweaks to get the same output as the file I put (which is simply changing the main Doxyfile.in, and generating the `usr` pdf documentation).